src/ubuntu/18.04/helix/arm64v8/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM arm64v8/ubuntu:18.04
+
+# Install Helix Dependencies
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get install -qq -y \
+        locales \
+        locales-all \
+        tzdata \
+        iputils-ping \
+        libunwind-dev \
+        libcurl4 \
+        gss-ntlmssp \
+        libgdiplus \
+        python-dev \
+        libunwind8 \
+        libssl-dev \
+        unzip \
+        libicu-dev \
+        libffi-dev \
+        build-essential \
+        cmake \
+        clang \
+        gdb \
+        lldb-3.9 \
+        git \
+        gcc \
+        python-pip \
+    && rm -rf /var/lib/apt/lists/* \
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
```

Comment:
From a readability perspective I think it would be good to add a blank line before this (e.g. `    \`) so that it doesn't get overlooked with the package installation/cleanup.